cli/src/commands/split.rs

```diff
@@ -51,9 +51,10 @@ pub(crate) struct SplitArgs {
     /// The revision to split
     #[arg(long, short, default_value = "@")]
     revision: RevisionArg,
-    /// Split the revision into two siblings instead of a parent and child.
-    #[arg(long, short)]
-    siblings: bool,
+    /// Split the revision into two parallel revisions instead of a parent and
+    /// child.
+    #[arg(long, short, alias = "siblings")]
```

Comment:
Maybe add a `TODO: remove after jj 0.25` or something so we can grep for it and find it.

Reply:
Done.